lib/routemaster/middleware/cache.rb

```diff
@@ -15,6 +16,7 @@ def initialize(app, cache:nil, client:nil, queue:nil)
 
       def call(env)
         env.fetch('routemaster.dirty', []).each do |url|
+          EventIndex.new(url).increment
```

Comment:
If I'm not mistaken, now that we have versioning, `@cache.bust` should not be required anymore?
(which is nice as one class fewer needs to know about Redis keys)